eskip/eskip.go

```diff
@@ -20,6 +20,20 @@ var (
 	duplicateMethodPredicateError   = errors.New("duplicate method predicate")
 )
 
+type DefaultFilters struct {
```

Comment:
A short godoc could explain that this is a PreProcessor and to be used with routing package.